gpustat/__main__.py

```diff
@@ -66,7 +71,27 @@ def main(*argv):
                         version=('gpustat %s' % __version__))
     args = parser.parse_args(argv[1:])
 
-    print_gpustat(**vars(args))
+    if args.interval > 0:
```

Comment:
It should be better to have a minimal interval of 0.1 sec like `watch`.

Reply:
Good idea.